### ISSUE TITLE
Formats missing the version field.

### ISSUE THREAD

**Issue by SquidDev** [oeed/CraftOS-Standards#60]
The following formats are missing a version field:
- [Temporary Directory](https://github.com/oeed/CraftOS-Standards/blob/master/Miscellaneous-Standards/Temporary%20Directory.md)
- [Directory](https://github.com/oeed/CraftOS-Standards/blob/master/File-Formats/application/directory.md) (this is in a different directory to the above - is this correct).
- NFT and Paint
- Lua and Plain text

I'd presume we'd want to use 1.0.0 for all of these? I'm happy to put together a PR once #58 is merged.

**Comment by oeed** [comment]
Yes 1.0.0 seems fine. The only mildly ambiguous one is Lua (i.e. do we use 5.1, etc.), but I guess that the version is more about the format itself – so 1.0.0 seems more appropriate. Open to discussion though.

**Comment by lyqyd** [comment]
There isn't anything I can see in lua.md that would require updating once ComputerCraft moves to Lua 5.2, so I'd say 1.0.0 is fine.

**Comment by viluon** [comment]
Lua versions are fundamentally different, and should be interpreted as separate languages really. Note that they don't follow [semantic versioning](http://semver.org), minor version number changes introduce backwards incompatibilities. Since the file extension doesn't change, however, I believe the _source format_ itself should have its own version number.

TLDR; 1.0.0 is the way to go!
